### PR TITLE
DB-3 Profile

### PR DESCRIPTION
**Pull request by rishano**
<h1>DB-3 Profile</h3>
<h2>Profile: </h4>
- Added dropdown menu for profile<br>
- Added animation to dropdown menu 

<h3> Tasklist Minor Fixes </h4>
- Fixed the font size of the tasklist to a bigger format to match the layout of the page<br>
- Fixed the divider line from the agenda to match the divider line of the tasklist and the stats image<br>
- Centered the header task of the tasklist and changed the font size of the profile name and the appointment date.<br>
- Added dropdown menu for profile<br>
- Made SCSS nested

### CODE REVIEW

File: Default/public/stylesheets/sass/style.scss [hunk 1]
```diff
@@ -424,42 +482,42 @@ body {
...
   color: #b7b7b7;
-  font-size: 12px;
+  font-size: 16px;
   float: right;
   margin-right: 39%;
-  margin-top: -8.5%;
+  margin-top: -12%;
 }
 
 .clockicon2 {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Should not exist. Should be .clockicon.

File: Default/public/stylesheets/sass/style.scss [hunk 2]
```diff
@@ -424,42 +482,42 @@ body {
...
   color: #b7b7b7;
-  font-size: 12px;
+  font-size: 16px;
   float: right;
   margin-right: 39%;
-  margin-top: -8.35%;
+  margin-top: -12%;
 }
 
 .tasklist__interface__appointmentdate {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Not BEM

File: Default/public/stylesheets/sass/style.scss [hunk 3]
```diff
@@ -479,15 +537,15 @@ body {
   margin-top: 7.7%;
   margin-left: 7%;
   color: #666666;
-  font-size: 57%;
+  font-size: 73%;
   font-family: arial;
 }
 
 .tasklist__interface__text2 {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Not BEM

File: Default/public/stylesheets/sass/style.scss [hunk 4]
```diff
@@ -506,9 +564,9 @@ body {
 
 .tasklist__interface__profilename2 {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Not BEM

File: Default/public/stylesheets/style.css [hunk 5]
```diff
@@ -334,23 +379,23 @@ body {
...
-  margin-top: -7%;
+  margin-top: -8.4%;
   margin-left: 7.5%;
   color: #a2a2a2;
-  font-size: 58%;
+  font-size: 69%;
   font-family: arial;
 }
 
 .tasklist__interface__profilename {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/style.css]: Not BEM

File: Default/public/stylesheets/style.css [hunk 6]
```diff
@@ -362,42 +407,42 @@ body {
...
   color: #b7b7b7;
-  font-size: 12px;
+  font-size: 16px;
   float: right;
   margin-right: 39%;
-  margin-top: -8.35%;
+  margin-top: -12%;
 }
 
 .tasklist__interface__appointmentdate {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/style.css]: Not BEM

File: Default/public/stylesheets/style.css [hunk 7]
```diff
@@ -417,15 +462,15 @@ body {
   margin-top: 7.7%;
   margin-left: 7%;
   color: #666666;
-  font-size: 57%;
+  font-size: 73%;
   font-family: arial;
 }
 
 .tasklist__interface__text2 {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/style.css]: Not BEM

File: Default/public/stylesheets/style.css [hunk 8]
```diff
@@ -444,9 +489,9 @@ body {
 
 .tasklist__interface__profilename2 {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/style.css]: Not BEM

File: Default/public/stylesheets/sass/style.scss (hunk 3 above)
Comment (rishano) [reply Default/public/stylesheets/sass/style.scss]: fixed

File: Default/public/stylesheets/sass/style.scss (hunk 2 above)
Comment (rishano) [reply Default/public/stylesheets/sass/style.scss]: fixed

File: Default/public/stylesheets/sass/style.scss (hunk 1 above)
Comment (rishano) [reply Default/public/stylesheets/sass/style.scss]: fixed

File: Default/public/stylesheets/sass/style.scss [hunk 9]
```diff
@@ -342,175 +400,182 @@ body {
...
-  height: 32%;
-  max-height: 100%;
-  max-width: 100%;
-  margin: 0 auto;
-  background: #5a5a5a;
-  border-radius: 7px;
-  border: solid 2px;
-  border-color: #f5f5f5;
-}
+  &__interface__header {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Geen BEM (&__interface-header)

File: Default/public/stylesheets/sass/style.scss [hunk 10]
```diff
@@ -342,175 +400,182 @@ body {
...
+  }
 
-.tasklist__interface__header {
-  margin-top: 7.7%;
-  margin-left: 7%;
-  color: #ffffff;
-  font-size: 66%;
-  font-family: arial;
-}
+  &__interface__text {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Geen BEM (&__interface-text)

File: Default/public/stylesheets/sass/style.scss [hunk 11]
```diff
@@ -342,175 +400,182 @@ body {
...
+    margin-left: 7.5%;
+    color: #a2a2a2;
+    font-size: 69%;
+    font-family: arial;
+  }
 
-.tasklist__interface__text {
-  margin-top: -7%;
-  margin-left: 7.5%;
+  &__interface__profilename {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Geen BEM (&__interface-profilename)

File: Default/public/stylesheets/sass/style.scss [hunk 12]
```diff
@@ -342,175 +400,182 @@ body {
...
+    margin-left: 22%;
+    margin-top: -8%;
+    color: #ffffff;
+    font-size: 80%;
+    font-family: arial;
+  }
+
+  &__interface__appointmentdate {
```
Comment (Giovanni-Schroevers) [inline Default/public/stylesheets/sass/style.scss]: Geen BEM (&__interface-appointmentdate)

**Pull request merged**